File: rest/domain/get-domain-mappings/get-domain-mappings.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example 
{
  static void Main(string[] args) 
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "AC32a3c49700934481addd5ce1659f04d2";
    string AuthToken = "{{ auth_token }}";
    var twilio = new TwilioRestClient(AccountSid, AuthToken);

    var ipAccessControlListMappings = twilio.ListIpAccessControlListMappings("SD32a3c49700934481addd5ce1659f04d2", null, null);
    
    foreach (var ipAccessControlListMapping in ipAccessControlListMappings.IpAccessControlListMappings)
    {
      Console.WriteLine(ipAccessControlListMapping.FriendlyName);
    }
  }
}

File: rest/credential-list-mapping/get-domain-mappings/get-domain-mappings.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'rubygems'          # This line not needed for ruby > 1.8
require 'twilio-ruby' 

# Get your Account Sid and Auth Token from twilio.com/user/account 
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = '{{ auth_token }}'
@client = Twilio::REST::Client.new account_sid, auth_token


# Loop over ip_access_control_list_mappings and print out a property for each one
@client.account.sip.domains.get('SD32a3c49700934481addd5ce1659f04d2').ip_access_control_list_mappings.list.each do |ip_access_control_list_mapping|
    puts ip_access_control_list_mapping.friendly_name
end
